ext/opentelemetry-ext-aiopg/src/opentelemetry/ext/aiopg/__init__.py

```diff
@@ -0,0 +1,132 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The integration with PostgreSQL supports the aiopg library,
+it can be enabled by using ``AiopgInstrumentor``.
+
+.. aiopg: https://github.com/aio-libs/aiopg
+
+Usage
+-----
+
+.. code-block:: python
+
+    import aiopg
+    from opentelemetry import trace
+    from opentelemetry.ext.aiopg import AiopgInstrumentor
+    from opentelemetry.sdk.trace import TracerProvider
+
+    trace.set_tracer_provider(TracerProvider())
```

Comment:
If you're going to have this, you will also need to include code to set up the exporter, right? Other exts seem to avoid the OT setup code so I would also recommend keeping this out
```suggestion
```

Reply:
done